### DISCUSSION TITLE
next/image cloudinary with prefix

### DISCUSSION THREAD

**Discussion by rohinrohin** [vercel/next.js#21919] [Help]
I am trying to integrate nextjs/image with the cloudinary loader. However, I am not able to add a prefix for the loader to fetch the images from the right folder from cloudinary. 

The final images live inside the following structure in cloudinary
dev > img > header.jpg

Repro: 
next.js.config
```
  module.exports.images = {
    loader: 'cloudinary',
     path: 'https://res.cloudinary.com/<cloudname>/dev/image/upload'
    //path: 'https://res.cloudinary.com/<cloudname>/dev'
   //tried both
  }
```

hello-world.js
```
import Image from 'next/image'

<Image 
    src='/img/header.jpeg'
    layout='fill'
    objectFit='contain'
    className='rounded-full'
/>
```

When I try to run the application here is how the image URLs look:
```
NextJs page: https://res.cloudinary.com/<cloudname>/dev/image/upload/f_auto,c_limit,w_1200,q_auto/img/header.jpg
```
However the expected URL should look like this as per cloudinary spec
```
Expected: https://res.cloudinary.com/<cloudname>/image/upload/f_auto,c_limit,w_1200,q_auto/dev/img/header.jpg
```

Been stuck on this for quite a bit and would appreciate any help from the community!

Thanks for your time!

BTW - will be using ``next export`` to host the site as a static page (image CDN and optimization via cloudinary)

**Comment by fe-linjin** [comment]
mark

**Comment by leerob** [comment]
@domitriusclark any thoughts?

**Comment by domitriusclark** [comment]
In your  `next.js.config` you have 'https://res.cloudinary.com/<cloudname>/dev/image/upload' 

I think the `dev` is what's breaking the URL here? When looking for a specific folder, it's usually apart of the `publicId`

Give 
```
src='dev/img/header.jpeg'
```
a try

can't tell too much without your cloudname (happy to dig further if that's not the case in a DM)
